internal/webhooks/vsphereclustertemplate_webhook.go

```diff
@@ -14,43 +14,58 @@ See the License for the specific language governing permissions and
 limitations under the License.
```

Comment:
Let's drop the _webhook suffixes from all files. Just unnecessary noise as the package name is now webhooks

Reply:
Renamed all the files in the webhooks package.